src/components/commands.component.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import Input from '@vini-vici/viddi/dist/input/input.component';
+
+export default function Commands(): React.ReactElement {
+  // Show the item?
+  const [show, setShow] = React.useState(false);
+  // loading commands
+  const [loading, setLoading] = React.useState(false);
+  // Searching
+  const [search, setSearch] = React.useState('');
+  // Keydown function
+  const keydownFn = (e: KeyboardEvent) => {
```

Comment:
For some reason now this is no longer working, even though when I committed these changes this worked as expected now suddenly the `show` variable is _always_ whatever the initial value is for the `useState` call. 

Found some references that are saying to use `useRef` to create a reference to the handler, but that doesn't appear to be changing anything -- command menu will pop up with `ctrl+k`, but it is currently impossible to make it toggle between shown and not shown as this code implies.

Reply:
This apparently seems to be related to the fact that I modified the code at the very end to properly include the `[]` in the `React.useEffect` hook so that it only ran on mount.